<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="LSTKTR242584" failures="1" tests="5" name="First Tests" time="0.047" errors="0" timestamp="2021-10-04T22:09:21 CDT">
  <testcase classname="Test.FirstTutorial" name="Demo" time="0.006">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Test.FirstTutorial.Demo(FirstTutorial.java:23)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:829)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- Demo -->
  <testcase classname="Test.SecondTutorial" name="ThirdTutorial" time="0.002"/>
  <testcase classname="Test.SecondTutorial" name="ploan" time="0.001"/>
  <testcase classname="Test.FourthTutorial" name="MobileLoginHome" time="0.002"/>
  <testcase classname="Test.FourthTutorial" name="WebLoginHome" time="0.001"/>
  <testcase classname="Test.FirstTutorial" name="SecondTest" time="0.0">
    <ignored/>
  </testcase> <!-- SecondTest -->
</testsuite> <!-- First Tests -->
